packages/fxa-auth-db-mysql/db-server/index.js

```diff
@@ -218,6 +218,18 @@ function createServer(db) {
 
   api.get('/securityEvents/:id/ip/:ipAddr', withParams(db.securityEvents));
   api.post('/securityEvents', withBodyAndQuery(db.createSecurityEvent));
+  api.get(
+    '/securityEvents/:id',
+    op(function(req) {
```

Comment:
I think I missed this in other PR, but lets use the new `=>` syntax where possible.